### PR TITLE
Custom 503 pages

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Added support for custom down for maintenance pages on disabled apps to fix issue #649 as suggested by Richard:
https://groups.google.com/d/msg/web2py-developers/guV2od3TV6c/T-7ZJX6dWe
EJ

### CODE REVIEW

File: gluon/main.py [hunk 1]
```diff
@@ -390,7 +390,11 @@ def wsgibase(environ, responder):
                                    % 'invalid request',
                                    web2py_error='invalid application')
                 elif not request.is_local and exists(disabled):
-                    raise HTTP(503, "<html><body><h1>Temporarily down for maintenance</h1></body></html>")
+                    if os.path.exists(os.path.join(request.folder,'static','503.html')):
+                        import urllib
+                        raise HTTP(503, urllib.urlopen(os.path.join(request.folder,'static','503.html')).read())
```
Comment (ilvalle) [inline gluon/main.py]: This doesn't work on python 3.5, please consider importing urlopen from pydal._compat

**Comment by niphlod** [comment]
please close this PR and issue one just for the issue at hand: updating each language file with the default translations doesn't belong here (and, frankly, isn't needed at all)

File: gluon/main.py (hunk 1 above)
Comment (niphlod) [inline gluon/main.py]: and why using urllib at all when you can read the file  ?!?!?

**Comment by vinyldarkscratch** [comment]
Oh that's weird, it shouldn't have requested to pull my second commit...  Oops, sorry about that!  (Still getting used to Git repositories and how they all work.)

For the use of URLLIB, I have no idea -- I literally just copied the code from the link I mentioned above and made a few slight changes.

File: gluon/main.py (hunk 1 above)
Comment (vinyldarkscratch) [inline gluon/main.py]: (I never saw these until just now...)

@niphlod You actually have a really good point...that's what I get for 4:30 AM code-writing.  XD

**Pull request closed without merging**